packages/main/test/pages/InputFieldLabels.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ui5-input</title>
+    <script src="../../bundle.esm.js" type="module"></script>
+</head>
+
+<style>
+    html,
+    body {
+        margin: 0;
+        height: 100%;
+        width: 100%;
+    }
+
+    body {
+        padding: 1rem;
+        box-sizing: border-box;
+    }
+
+    .wrapper {
+        padding: 0.5rem;
+        box-sizing: border-box;
+        border: 1px grey dashed;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+    }
+
+    .field {
+        margin-top: 0.25rem;
+        display: block;
+        max-width: 100%;
+    }
+
+    .field,
+    .fields {
+        width: 540px;
+        max-width: 100%;
+    }
+</style>
+
+<body>
+    <div class="wrapper">
+        <div class="fields">
+            <p>
+                <ui5-label id="lblDesc1" for="myInput">Description that will change</ui5-label>
+            </p>
+            <p>
+                <ui5-label id="lblDesc2" for="myInput">Another description</ui5-label>
+            </p>
+            <p>
+                <ui5-input id="myInput" placeholder="input placeholder" class="field"></ui5-input>
+            </p>
+            <p>
+                <ui5-label id="lblDesc3" for="myInput">
+                    Description that will be removed</ui5-label>
+            </p>
+            <p><label for="myInput">DescLabel</label></p>
+            <p><ui5-button id="btnChange">Change Desc</ui5-button></p>
+            <p id="actualLabel">Actual Desc from input:</p>
+            <hr />
+            <p>
+                lblEnterName1 &nbsp;<ui5-label id="lblEnterName1">First Label Desc</ui5-label>
+            </p>
+            <p>
+                lblEnterName2 &nbsp; <ui5-label id="lblEnterName2">Second Label Desc</ui5-label>
+            </p>
+            <p>
+                <ui5-input id="inputEnterName" accessible-name-ref="lblEnterName1" placeholder="Enter your name"
```

Comment:
You can also check the case with multiple referenced labels by `accessible-name-ref` to make sure it works in all cases

```html
<ui5-label id="text">label 1</ui5-label>
<ui5-label id="text2">label 2</ui5-label>
<ui5-input accessible-name-ref="text text2"></ui5-input>
```